src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1649,8 +1649,22 @@ public sealed class Options : BinaryOptionsBase
             /// <value>
             /// If unspecified, <see cref="LogLoss"/> will be used.
             /// </value>
-            [Argument(ArgumentType.Multiple, HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
-            public ISupportSdcaClassificationLossFactory LossFunction = new LogLossFactory();
+            [Argument(ArgumentType.Multiple, Name = "LossFunction", HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
+            internal ISupportSdcaClassificationLossFactory LossFunctionFactory = new LogLossFactory();
+
+            /// <summary>
+            /// The custom <a href="tmpurl_loss">loss</a>.
+            /// </summary>
+            /// <value>
+            /// If unspecified, <see cref="LogLoss"/> will be used.
+            /// </value>
+            public ISupportSdcaClassificationLoss LossFunction;
+
+            public Options()
```

Comment:
Please add user-facing document. Thanks.

Reply:
constructor removed

---
In reply to: [262724236](https://github.com/dotnet/machinelearning/pull/2856#discussion_r262724236) [](ancestors = 262724236)